api/organisations/models.py

```diff
@@ -212,6 +220,7 @@ def update_mailer_lite_subscribers(self):
 
     def cancel(self, cancellation_date=timezone.now(), update_chargebee=True):
         self.cancellation_date = cancellation_date
+        self.billing_status = ""
```

Comment:
I kind of prefer `None` here to indicate no status but it's a minor nitpicky preference.

Reply:
Oh yeah, me too. Sorry for the oversight, `None` is definitely the way to go, although we'll still get `""` in the db due to Django admin sending empty strings.